platform-dependent-components/problem-solver/py/server.py

```diff
@@ -5,12 +5,12 @@
 """
 
 import argparse
-
-from modules.clean_text_generation_module.clean_text_generation_module import \
-    CleanTextGenerationModule
-from modules.example_module.example_module import ExampleModule
 from sc_kpm import ScServer
 
+from modules.exampleModule.ExampleModule import ExampleModule
```

Comment:
ты поменял из правильного модуля в неправильный. Верни импорты как было, только добавь один свой, зачем другие трогать?
```suggestion
from modules.example_module.example_module import ExampleModule
```

Reply:
@MksmOrlov fixed